src/coap/coapserver.c

```diff
@@ -90,7 +82,9 @@ int coapserver_listen(uint16_t sport, recv_handler_t recv_handler)
 
   m_sockfd = sockfd;
   m_server_opened = true;
-  recvt_id_task = osal_task_create(NULL, 0, 0, recv_thread, NULL);
+  ret = osal_task_create(&recvt_id_task, NULL, 0, 0, recv_thread, NULL);
+  assert(ret == 0);
+
   return 0;
```

Comment:
Return values to be replaced by appropriate macros in all instances.

Reply:
Related to our previous discussion and recommendations

Reply:
Yes, all such instances could be replaced with valid defines. Thanks.